doc/OnlineDocs/source/tasks/install.rst

```diff
@@ -0,0 +1,89 @@
+Installation
+============
+
+Prescient is a python package with a number of dependencies and prerequisites.
+
+To install Prescient, follow these steps:
+
+.. contents::
+   :local:
+
+
+Install python
+--------------
+Prescient requires python 3.7 or later. We recommend installing `Anaconda <https://www.anaconda.com>`_
+to manage python and other dependencies.
+
+
+Get Prescient source code
+-------------------------
+The latest stable version of Prescient can be acquired as source from
+`the Prescient github project <https://github.com/grid-parity-exchange/Prescient>`_,
+either by downloading a zip file of the source code or by cloning the `main` branch of
+the github repository.
+
+
+Install dependencies
+--------------------
+Prescient runs in a python environment that must include a number of python prerequisites.
+You may want to create a python environment specifically for Prescient. To create a new
+Anaconda environment that includes Prescient's prerequisites, issue the following command
+from the root folder of the Prescient source code::
+
+	conda env create -f environment.yml
+
+The command above will create an environment named `prescient`. To use a different name for the
+environment, add the `-n` option to the command above::
+
+	conda env create -n nameOfYourChoice -f environment.yml
+
+Once you have create the new environment, make it the active environment::
+
+    conda activate prescient
+
+If you are using something other than Anaconda to manage your python environment, use the 
+information in `environment.yml` to identify which packages to install. 
+
+
+Install a linear solver
+-----------------------
+Prescient requires a mixed-integer linear programming (MILP) solver that is compatible with
+`Pyomo <https://pyomo.readthedocs.io>`_. Options include open source solvers such as CBC, GLPK,
+or SCIP, and commercial solvers such as CPLEX, Gurobi, or Xpress.

```

Comment:
I don't think we actually support SCIP..

Reply:
It's listed in the README.md. I'm fine taking it out.